README.md

```diff
@@ -196,12 +197,15 @@ Run `tns help` to list all available commands. Run or `tns <Command> --help` to
 * `platform list` lists the current target platforms for your project.
 * `platform add <Platform>` adds a new target platform to your project.
 * `platform remove <Platform>` removes the selected platform from the target platforms of the project.
+* `platform update <Platform>` updates the NativeScript runtime for the specified platform.
 * `prepare <Platform>` copies cross-platform and selected platform-specific content to the subdirectory for the target platform.
 * `build <Platform>` builds the project for the selected target platform.
 * `emulate <Platform>` builds the project for the selected target platform and runs it in the native emulator, if configured. 
 * `deploy <Platform> [--device <Device ID>]` deploys an already built application on connected device.
 * `run <Platform> [--device <Device ID>]` executes `prepare`, `build`, and `deploy`.
-* `list-devices` lists connected devices, including any running Android Virtual Devices.
+* `device` lists connected devices, including any running Android Virtual Devices or Genymotion virtual devices.
+* `device run` runs a selected application on a connected device.
+* `device list-applications` lists the installed applications on all connected devices.
```

Comment:
Is it working for the NS CLI? I'm asking because it's not in the help.txt at all :)